TrackingTools/TransientTrack/src/TransientTrackBuilder.cc

```diff
@@ -91,6 +93,91 @@ TransientTrackBuilder::build (const edm::Handle<edm::View<Track> > & trkColl) co
 
 vector<TransientTrack> 
 TransientTrackBuilder::build ( const edm::Handle<reco::TrackCollection> & trkColl,
+			       edm::Handle<edm::ValueMap<float> > trackTimesH,
+			       edm::Handle<edm::ValueMap<float> > trackTimeResosH ) const
+{
+  vector<TransientTrack> ttVect;
+  ttVect.reserve((*trkColl).size());
+  const auto& trackTimes = *trackTimesH;
+  const auto& trackTimeResos = *trackTimeResosH;
+  for (unsigned int i = 0; i < (*trkColl).size() ; i++) {
+    TrackRef ref(trkColl, i);
+    double time = trackTimes[ref];
+    double timeReso = trackTimeResos[ref];
+    timeReso = ( timeReso > 1e-6 ? timeReso : 0.170 ); // make the error much larger than the BS time width
```

Comment:
is this magic 0.170 in a relationship with 0.175 used in another file in this PR?
Better be consistent and also use some named constant